src/services/auth_service.js

```diff
@@ -0,0 +1,15 @@
+export default class AuthService {
+  static verifyAuth() {
+    if (localStorage.getItem('access_token')) {
+
+    } else {
+      AuthService.authenticateThroughRideService()
+    }
+  }
+
+  static authenticateThroughRideService() {
+    
```

Comment:
A comment here would be excellent

Reply:
That was intended for refactoring. I will remove it for now.